ci/playbooks/deploy_podified_ctlplane.yaml

```diff
@@ -0,0 +1,81 @@
+---
+- hosts: controller
+  tasks:
+    - name: Install Ansible, Make, Tar
+      become: true
+      ansible.builtin.package:
+        name:
+          - ansible-core
+          - make
+          - tar
+        state: latest
+
+    - name: Install devsetup dependencies
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/devsetup"
+        target: download_tools
+
+    - name: Create OpenStack namespace
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: namespace
+        params: &install_yamls_out
+          OUT: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/out"
+
+    - name: Generate olm artifacts
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: openstack_prep
+        params: *install_yamls_out
+
+    - name: Deploy Openstack Operators
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: openstack
```

Comment:
openstack depends on openstack_prep and namesapce as well as input

make openstack depens on everything it requires excpet crc_stroage which is being provided differntly

Reply:
Removed extra invocations.